Scripts/cmake/FindiPlug2.cmake

```diff
@@ -292,12 +291,6 @@ elseif (CMAKE_SYSTEM_NAME MATCHES "Linux")
   list(APPEND _opts "-Wno-multichar")
 
 elseif (CMAKE_SYSTEM_NAME MATCHES "Darwin")
-  list(APPEND _src 
-    ${IPLUG_SRC}/IPlugPaths.mm

```

Comment:
Sorry for taking a while to look at this. Shouldn't IPlugPaths.mm still be included ?